geode-docs/tools_modules/pulse/pulse-auth.html.md.erb

```diff
@@ -50,3 +50,95 @@ When the `http` SSL component is enabled, all HTTP services become
 SSL-enabled and you must configure your client applications
 accordingly. For SSL-enabled Pulse, you will need to configure your
 browsers with proper certificates.
+
+# Configuring Pulse to Use Geode Security Manager for Authentication/Authorization
+
+**Embedded Mode (Jetty)**
+
+You don't need to do anything for this setup. Pulse will automatically be configured by Geode to send the credentials entered in Pulse login page to the Security Manager for authentication and authorization.
+
+**Non-Embedded (Standalone Web Server) Mode (Tomcat)**
+
+You need to configure the Pulse web app to use a geode specific security profile by activating the geode profile at startup with the system property:
+
+``` pre
+-Dspring.profiles.active=pulse.authentication.gemfire
+```
+
+# Configuring Pulse to use Custom Security Profile
+
+This would only work if you are NOT using a security manager to in your cluster. The custom security profile only controls who can access Pulse web page. You will need to create a xml file named `pulse-authentication-custom.xml` that defines the credentials and roles of the users of Pulse.
+
+**Example pulse-authentication-custom.xml for Spring simple in-memory User Details Service**
+
+``` pre
+<beans:beans >
+  <authentication-manager>
+    <authentication-provider>
+      <user-service id="userDetailsService">
+        <user name="john" password="johnspassword" authorities="ROLE_USER " />
```

Comment:
Are these authorities correct? I would have guessed that they would have to something `CLUSTER:READ`.

If `ROLE_USER` is valid here, okay. But if not, then consider giving John and Bob real authorities. And perhaps give each a different set of authorities.